### ISSUE TITLE
Role image alt text does not work correctly

### ISSUE THREAD

**Issue by JoshHash** [dnldev/Avaclon#31]
When switching between hiding and showing the users role the alt text disappears, making it impossible to switch in that time.
![image](https://user-images.githubusercontent.com/37668961/40618603-4c9298e0-6292-11e8-8143-e50644a649a7.png)

The alt text always shows the role even in a hidden state.
![image](https://user-images.githubusercontent.com/37668961/40618617-5a4d96d8-6292-11e8-8d8f-8616e0f51664.png)
